### ISSUE TITLE
Bug: ColorVariable hex vs. rgb

### ISSUE THREAD

**Issue by marcobiedermann** [brigade/scss-lint#413]
The following code results in an warning.

```
$color-bg: rgba(255, 255, 255, .8); // valid
$color-bg: rgba(#fff, .8);          // invalid
```

Error: `ColorVariable: Color literals like #fff should only be used in variable declarations; they should be referred to via variable everywhere else.`

I think this is a bug because I am using a variable to set the color. It shout not matter whether using hex or rgb as color format

**Comment by sds** [comment]
Hey @marcobiedermann, thanks for the report. 

This was reported in #402 and fixed in 4b91c9e6a73a8bb0d47ac2789a162133a798a908. In order to prevent further confusion, I've released a new version ([0.36.0](https://github.com/brigade/scss-lint/releases/tag/v0.36.0)) of `scss-lint` with the fix.

**Comment by marcobiedermann** [comment]
@sds Oh, thank you very much. I've taken a look at the closed issues but did not find it.
